src/index.js

```diff
@@ -4,7 +4,7 @@ import './css/base.scss';
 import User from './User'
 import Bookings from './Bookings'
 
-let currentUser, bookings, userBookings;
+let currentUser, bookings, userBookings, selectedDate;
 let todaysDate = new Date();
 let dd = String(todaysDate.getDate()).padStart(2, '0');
 let mm = String(todaysDate.getMonth() + 1).padStart(2, '0');
```

Comment:
You may want to eventually make line 10 more dynamic by writing a function only adding a "0" when necessary. I'm not 100% sure I'm reading that correctly though.